### PR TITLE
[Reviewer: Phil] Allow multiple users to run stress at once

### PR DESCRIPTION
**Pull request by rkday-pro**
(by moving from fixed filenames to per-process ones)

### CODE REVIEW

File: clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress [hunk 1]
```diff
@@ -52,15 +54,17 @@ class SIPpstats(csv.excel):
 SIPP_BIN = "/usr/share/clearwater/bin/sipp_coreonly"
 SCRIPTS_DIR = "/usr/share/clearwater/sip-stress/"
 
-REGISTER_CSV = "/tmp/registers.csv"
-INVITE_CSV = "/tmp/calls.csv"
+REGISTER_CSV = NamedTemporaryFile(prefix="sipp_register")
+INVITE_CSV = NamedTemporaryFile(prefix="sipp_calls")
```
Comment (pdavies) [inline clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress]: Niggle: maybe these shouldn't be ALL_CAPS now they're objects, not constants?

Comment (pdavies) [inline clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress]: You should probably pass in `delete=False` here. The files are dangerously close to getting deleted automatically (in particular, if someone ever changes the script to close the file handles with `.close()`, the files will suddenly vanish).

File: clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress [hunk 2]
```diff
@@ -278,4 +282,4 @@ Average time from REGISTER to 200 OK: {} ms""".format(
     row['Retransmissions(C)'],
     rtt)
 
-print "\nLog files in /var/log/clearwater-sip-stress"
+print "\nLog files in /var/log/clearwater-sip-stress, with prefix {}".format(os.getpid())
```
Comment (pdavies) [inline clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress]: Is this right? Earlier, the prefix was either "sipp_register" or "sipp_calls".

**Review by pdavies — APPROVED** [review]

File: clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress (hunk 1 above)
Comment (rkday-pro) [reply clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress]: I want that behaviour, actually - we don't want these CSV files cluttering up the disk, so using a NamedTemporaryFile this way means they get automatically deleted when the filehandles are closed at program exit.

File: clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress (hunk 2 above)
Comment (rkday-pro) [reply clearwater-sip-stress-coreonly.root/usr/share/clearwater/bin/run_stress]: This relate to the prefix at line 60 (where we prefix our error logs with our PID) - I'll use LOG_PREFIX directly to make the link clearer.

**Pull request merged**